scripts/globals/abilities/chocobo_jig.lua

```diff
@@ -20,6 +20,11 @@ abilityObject.onUseAbility = function(player, target, ability)
     local durationMultiplier = 1.0 + utils.clamp(player:getMod(xi.mod.JIG_DURATION), 0, 50) / 100
     local finalDuration      = math.floor(baseDuration * durationMultiplier)
 
+    -- TODO: Chocobo Jig will not override all types of weight effect. Find out which aren't overriden.
```

Comment:
probably a power/subpower/tier value needs to be involved for resolving that TODO later